ls1intum/Artemis
# [Bugfix] [Exam] StudentExam Detail View

### Checklist

### Motivation and Context
Loads of:
`ERROR TypeError: Cannot read property '0' of undefined`
were thrown in the student exam detail view if the results were not present.

### Description
Add check if student participation is present before trying to access its properties

### Steps for Testing
1. Log in to Artemis
2. Navigate to an exam with registered users and generated exams, but no assessments/results
3. The view should load normally, list all exercises and no console errors should be shown

### Screenshots

![image](https://user-images.githubusercontent.com/15110960/94132486-af065f00-fe5f-11ea-9b5b-4309c0e4eb0a.png)
![image](https://user-images.githubusercontent.com/15110960/94132596-ccd3c400-fe5f-11ea-917c-0daa7e0be35b.png)
